tomplot/local_area_plot.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
+
+
+def area_restriction(field_data, coords_X, coords_Y, Y_lim=None, X_lim=None):
+    """
+    a function that takes a data dictionary and will restrict it for a given value
+    For the coordinate fields you must pass in a 1d object so either coords[:,level]
+    for slicing on z or coords.flatten if slicing along lat / lon
+
+    args:
+    field_data: The Plotting field you wish to return
```

Comment:
The style of these docstrings isn't quite right. The style I use is the google style, see e.g. https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e

Three main things you need to do:
- indent your arguments
- add the type of the argument in brackets, before the colon
- for the comments, it's best to use a passive tense (and not use "you wish")

So this should be something like:
```
         field_data (`:class:`numpy.ndarray`): the field to be filtered.
```

You also have a rogue capital letter here!